### PR TITLE
Add missing language identifiers

### LINKED ISSUE #2192: Supply missing language identifiers to code blocks

### PR DESCRIPTION
**Pull request by bradriske**
## Summary

Add language identifiers to code blocks specified

Fixes #2192

### CODE REVIEW

File: docs/framework/configure-apps/file-schema/wcf/add-of-baseaddressprefixfilter.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Represents a configuration element that specifies a pass-through filter, which p
   
  For example, your site can contain the following base addresses:
   
-```
+```text
```
Comment (Youssef1313) [inline docs/framework/configure-apps/file-schema/wcf/add-of-baseaddressprefixfilter.md:52]:
```suggestion
```http
```

File: docs/framework/configure-apps/file-schema/wcf/baseaddressprefixfilters.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ Represents a collection of configuration elements that specify pass through filt
   
  For example, your site can contain the following base addresses:
   
-```
+```text
```
Comment (Youssef1313) [inline docs/framework/configure-apps/file-schema/wcf/baseaddressprefixfilters.md:54]:
```suggestion
```http
```

File: docs/framework/wcf/feature-details/messaging-protocols.md [hunk 3]
```diff
@@ -331,7 +331,7 @@ HTTP Headers: The destination in POST matches the URI in the `wsa10:To` element.
 
 The Content-Type header has the value of `application/soap+xml` as required by SOAP 1.2. Parameters `charset` and `action` are included. The `action` parameter of the Content-Type header matches the value of the `wsa10:Action` message header.
 
-```
+```text
```
Comment (Youssef1313) [inline docs/framework/wcf/feature-details/messaging-protocols.md:334]:
```suggestion
```http
```

File: docs/framework/wcf/feature-details/messaging-protocols.md [hunk 4]
```diff
@@ -358,7 +358,7 @@ Proxy-Connection: Keep-Alive
 
 The receiver responds with an empty HTTP response and status 202. An example of the HTTP response:
 
-```
+```text
```
Comment (Youssef1313) [inline docs/framework/wcf/feature-details/messaging-protocols.md:361]:
```suggestion
```http
```

File: docs/framework/wcf/feature-details/messaging-protocols.md [hunk 5]
```diff
@@ -574,7 +574,7 @@ MTOM is an encoding mechanism that is similar to `text/xml` and WCF Binary XML.
 
 #### WCF SOAP 1.1 Message Encoded Using MTOM
 
-```
+```text
```
Comment (Youssef1313) [inline docs/framework/wcf/feature-details/messaging-protocols.md:577]:
```suggestion
```http
```

File: docs/framework/wcf/feature-details/messaging-protocols.md [hunk 6]
```diff
@@ -608,7 +608,7 @@ Content-Type: application/octet-stream
 #### WCF Secure SOAP 1.2 Message Encoded Using MTOM
 In this example, a message is encoded using MTOM and SOAP 1.2 that is protected using WS-Security. The binary parts identified for encoding are the contents of the `BinarySecurityToken`, `CipherValue` of the `EncryptedData` corresponding to the encrypted signature and encrypted body. Note that the `CipherValue` of the `EncryptedKey` was not identified for optimization by WCF, because its length is less then 1024 bytes.
 
-```
+```text
```
Comment (Youssef1313) [inline docs/framework/wcf/feature-details/messaging-protocols.md:611]:
```suggestion
```http
```

File: docs/framework/wcf/feature-details/workflow-services-overview.md [hunk 7]
```diff
@@ -54,7 +54,7 @@ Workflow services hosted in a managed .NET Framework application or a managed Wi
 
 To host a workflow service in IIS or WAS, place the .xamlx file that contains the workflow service definition into a virtual directory. A default endpoint (using <xref:System.ServiceModel.BasicHttpBinding>) is created automatically For more information, see [Simplified Configuration](../simplified-configuration.md). You can also place a Web.config file in the virtual directory to specify your own endpoints. If your workflow definition is in an assembly you can place a .svc file in the virtual directory and the workflow assembly in the App_Code directory. The .svc file must specify the service host factory and the class that implements the workflow service. The following example shows how to specify the service host factory and specify the class that implements the workflow service.
 
-```
-<%@ServiceHost Factory=" System.ServiceModel.Activities.Activation.WorkflowServiceHostFactory
+```text
```
Comment (Youssef1313) [inline docs/framework/wcf/feature-details/workflow-services-overview.md:57]:
```suggestion
```aspx-csharp
```

File: docs/framework/wpf/advanced/globalization-for-wpf.md [hunk 8]
```diff
@@ -19,13 +19,13 @@ This topic introduces issues that you should be aware of when writing [!INCLUDE[
 ### Character References
 A character reference gives the UTF16 code unit of the particular Unicode character it represents, in either decimal or hexadecimal. The following example shows a decimal character reference for the COPTIC CAPITAL LETTER HORI, or 'Ϩ':
 
-```
+```xaml
```
Comment (Youssef1313) [inline docs/framework/wpf/advanced/globalization-for-wpf.md:22]: I don't think this is xaml

File: samples/snippets/core/testing/unit-testing-using-dotnet-test/csharp/README.md [hunk 9]
```diff
@@ -10,7 +10,7 @@ This sample demonstrates creating a library and writing effective unit tests tha
 
 To run the tests, navigate to the *PrimeService.Tests* directory and type the following commands:
 
-```
+```console
```
Comment (Youssef1313) [inline samples/snippets/core/testing/unit-testing-using-dotnet-test/csharp/README.md:13]:
```suggestion
```dotnetcli
```

File: samples/snippets/core/testing/unit-testing-using-mstest/csharp/README.md [hunk 10]
```diff
@@ -10,7 +10,7 @@ This sample demonstrates creating a library and writing effective unit tests tha
 
 To run the tests, navigate to the *PrimeService.Tests* directory and type the following commands:
 
-```
+```console
```
Comment (Youssef1313) [inline samples/snippets/core/testing/unit-testing-using-mstest/csharp/README.md:13]:
```suggestion
```dotnetcli
```

File: samples/snippets/core/testing/unit-testing-using-nunit/csharp/README.md [hunk 11]
```diff
@@ -10,7 +10,7 @@ This sample demonstrates creating a library and writing effective unit tests tha
 
 To run the tests, navigate to the *PrimeService.Tests* directory and type the following commands:
 
-```
+```console
```
Comment (Youssef1313) [inline samples/snippets/core/testing/unit-testing-using-nunit/csharp/README.md:13]:
```suggestion
```dotnetcli
```

File: samples/snippets/csharp/classes-quickstart/README.md [hunk 12]
```diff
@@ -12,7 +12,7 @@ This sample demonstrates introduction of object oriented programming with classe
 
 To build and run the sample, type the following two commands:
 
-```
+```console
```
Comment (Youssef1313) [inline samples/snippets/csharp/classes-quickstart/README.md:15]:
```suggestion
```dotnetcli
```

File: samples/snippets/csharp/delegates-and-events/README.md [hunk 13]
```diff
@@ -14,7 +14,7 @@ using multicast delegates.
 
 To build and run the sample, type the following two commands:
 
-```
+```console
```
Comment (Youssef1313) [inline samples/snippets/csharp/delegates-and-events/README.md:17]:
```suggestion
```dotnetcli
```

File: samples/snippets/csharp/events/README.md [hunk 14]
```diff
@@ -13,7 +13,7 @@ events, and release event handlers.
 
 To build and run the sample, type the following two commands:
 
-```
+```console
```
Comment (Youssef1313) [inline samples/snippets/csharp/events/README.md:16]:
```suggestion
```dotnetcli
```

File: samples/snippets/csharp/getting-started/console-linq/README.md [hunk 15]
```diff
@@ -10,7 +10,7 @@ This sample demonstrates querying data sources and processing them using Languag
 
 To build and run the sample, type the following two commands:
 
-```
+```console
```
Comment (Youssef1313) [inline samples/snippets/csharp/getting-started/console-linq/README.md:13]:
```suggestion
```dotnetcli
```

File: samples/snippets/csharp/getting-started/console-webapiclient/README.md [hunk 16]
```diff
@@ -13,7 +13,7 @@ and `await`, converting JSON objects into C# objects, and terminal output.
 
 To build and run the sample, type the following two commands:
 
-```
+```console
```
Comment (Youssef1313) [inline samples/snippets/csharp/getting-started/console-webapiclient/README.md:16]:
```suggestion
```dotnetcli
```

File: samples/snippets/csharp/versioning/new/README.md [hunk 17]
```diff
@@ -13,7 +13,7 @@ the `new` modifier when versioning .NET libraries.
 
 To build and run the sample, type the following two commands:
 
-```
+```console
```
Comment (Youssef1313) [inline samples/snippets/csharp/versioning/new/README.md:16]:
```suggestion
```dotnetcli
```

File: samples/snippets/csharp/versioning/override/README.md [hunk 18]
```diff
@@ -13,7 +13,7 @@ the `override` modifier when versioning .NET libraries.
 
 To build and run the sample, type the following two commands:
 
-```
+```console
```
Comment (Youssef1313) [inline samples/snippets/csharp/versioning/override/README.md:16]:
```suggestion
```dotnetcli
```

**Review by Youssef1313 — CHANGES_REQUESTED** [review]
I've suggested changes where I *think* there is a more appropriate identifier.
If you're unsure of a specific suggestion, we can wait for an official review.

**Review by tdykstra — COMMENTED** [review]
Thanks for this PR. Unfortunately, "text" and "conf" are not valid identifiers. We don't have a good choice for text yet, which is why many of these had been left blank.  The closest is "output", which doesn't fit many of those that are actually just text blocks. I recommend leaving the identifier off text blocks that aren't showing CLI output. At some future date a build validation will require identifiers, and hopefully by then we will have something like "text" available.

**Comment by bradriske** [comment]
Applied the suggestions and removed the ```conf entries.  After the latest commit there are still 152 lines of ```text in 85 .md files across the repo, should those all get changed to ``` or just leave as is for the later add to allow text type?

**Comment by tdykstra** [comment]
I would omit the "text" identifier. However, some of the blocks labeled "text" (and some that got labeled "console") are actually showing console output, and for those, "output" is a valid identifier that could be used and would accurately describe the content.

**Comment by tdykstra** [comment]
It looks like the PR still changes a number of blocks to text, some to console that could be output, and a couple to console that could be left blank.

**Comment by bradriske** [comment]
That should be it this time

**Review by tdykstra — APPROVED** [review]
Looks good, thanks!

**Pull request merged**